syntax = "proto3";

package GFSClient;

message AcknowledgeDataReceiptRequest {
  int32 write_id = 1;
}

message AcknowledgeDataReceiptResponse {
  bool acknowledgement_accepted = 1;
}

service GFSClientService {
  rpc AcknowledgeDataReceipt(AcknowledgeDataReceiptRequest) returns (AcknowledgeDataReceiptResponse) {}
}
